auth-web/src/components/pay/eft/ShortNameTransactions.vue

```diff
@@ -0,0 +1,168 @@
+      :setItems="state.results"
+      :setHeaders="headers"
+      :setTableDataOptions="tableDataOptions"
+      :totalItems="state.totalResults"
+      :filters="state.filters"
+      :updateFilter="updateFilter"
+      :pageHide="true"
+      @update-table-options="tableDataOptions = $event"
+    >
+      <template #header-filter-slot />
+      <template #item-slot-transactionDate="{ item }">
+        <span>{{ formatDate(item.transactionDate, 'MMMM DD, YYYY') }}</span>
+      </template>
+      <template #item-slot-depositAmount="{ item }">
+        <span>{{ formatCurrency(item.depositAmount) }}</span>
+      </template>
+    </BaseVDataTable>
+  </v-card>
+</template>
+<script lang="ts">
+import { Ref, defineComponent, reactive, ref, watch } from '@vue/composition-api'
+import { BaseVDataTable } from '@/components'
+import CommonUtils from '@/util/common-util'
+import { DEFAULT_DATA_OPTIONS } from '../../datatable/resources'
+import { DataOptions } from 'vuetify'
+import { EFTTransactionFilterParams } from '@/models/eft-transaction'
+import PaymentService from '@/services/payment.services'
+import _ from 'lodash'
+
+export default defineComponent({
+  name: 'ShortNameTransactions',
+  components: { BaseVDataTable },
+  props: {
+    shortName: {
+      type: Object,
+      default: () => ({
+        shortName: null,
+        transactionDate: null
+      })
+    }
+  },
+  setup (props) {
+    const headers = [
+      {
+        col: 'transactionDate',
+        hasFilter: false,
+        minWidth: '125px',
+        value: 'Payment Received Date'
+      },
+      {
+        col: 'depositAmount',
+        hasFilter: false,
+        minWidth: '125px',
+        value: 'Amount'
+      }
+    ]
+
+    const state = reactive({
+      results: [],
+      totalResults: 1,
+      filters: {
+        isActive: false,
+        pageNumber: 1,
+        pageLimit: 5
+      } as EFTTransactionFilterParams,
+      loading: false,
+      shortName: {
+        id: null
+      }
+    })
+
+    function setShortName (shortname) {
+      state.shortName = shortname
+      loadTransactions(state.shortName.id)
+    }
+
+    watch(() => props.shortName, () => setShortName(props.shortName), { deep: true })
+    async function loadTransactions (shortnameId: string): Promise<void> {
+      try {
+        state.loading = true
+        const response = await PaymentService.getEFTTransactions(shortnameId, state.filters)
+        if (response?.data) {
+          /* We use appendToResults for infinite scroll, so we keep the existing results. */
+          state.results = response.data.items
+          state.totalResults = response.data.total
+        } else {
+          throw new Error('No response from getEFTTransactions')
+        }
+      } catch (error) {
+        // eslint-disable-next-line no-console
+        console.error('Failed to getEFTTransactions list.', error)
+      }
+      state.loading = false
+    }
+
+    function updateFilter () : void { }
```

Comment:
I don't think we need?

Reply:
Removed, after merge with main that provides a default